syntax = "proto3";

import "types.proto";

service Gardens {
    rpc CreateGarden (garden.CreateGardenRequest) returns (garden.CreateGardenResult);
    rpc GetGardens (garden.GetGardensRequest) returns (stream garden.GetGardensResult);
}